lora.cpp

```diff
@@ -186,19 +194,6 @@ void lora::sendUplink(char *payload)
       DEBUG_SERIAL_LOG.printf("Sending request attempt %i\r\n", i);
```

Comment:
As mentioned in a different comment, it seems that they discourage the use of `delay`. Is there another way to do this that is still energy-efficient enough for the battery-powered version?

Reply:
I was correct. I have removed the loop altogether.